sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -4,8 +4,10 @@
 # license information.
 # -------------------------------------------------------------------------
 import time
-from typing import TYPE_CHECKING, Dict, Optional, TypeVar
-
+from typing import TYPE_CHECKING, Optional, TypeVar, MutableMapping, Any
+from azure.core.pipeline import PipelineRequest, PipelineResponse
+from azure.core.pipeline.transport import HttpResponse, HttpRequest
```

Comment:
Alias them to Legacy for clarity, and keep the rest one with their name:
https://github.com/Azure/azure-sdk-for-python/blob/c15c08437edc64662ff759572b99f7c0ba46ac78/sdk/core/azure-core/azure/core/polling/base_polling.py#L50-L56